domain/src/main/java/com/teamagam/gimelgimel/domain/messages/DisplayMessagesInteractor.java

```diff
@@ -55,12 +56,7 @@
                 this::transformToPresentation,
                 mDisplayer::show);
 
-        DisplaySubscriptionRequest updateMapDisplayStatusChanges = factory.create(
```

Comment:
I think it was through the interactor. I checked with Bar.
It was duplicate code calling of mDisplayer::show, it means that for every change, we notify for new message.